contracts/interstake-yield-generator/src/error.rs

```diff
@@ -48,6 +48,9 @@ pub enum ContractError {
     #[error("Commission address may not be the same as the recipient")]
     CommissionAddressSameAsRecipient {},
 
+    #[error("Unbonding called too soon after last unbonding")]
+    UnbondingTooSoon {},
```

Comment:
This will be used when we go over max entries limit, right? 
I'd rather rephrase that - `Unbonding max_limit exceeded`, please try next day or something?

Reply:
Okay, I've read rest of implementation. My suggestion doesn't make sense in this context, but I'd still provide at least some info about implementation and what does it mean "too soon" etc.

Reply:
Definitely. I rephrased the error name and message, to include some more info